joshuaCarpentier/assignments/fastSlow/lc141/lc141.js

```diff
@@ -35,24 +81,6 @@ head.next.next.next.next.next.next = head.next.next.next
 console.log(`LinkedList has cycle: ${has_cycle(head)}`)
 
 
-
-
-// Solution
-// -----
-// function has_cycle(head) {
-//   let slow = head,
-//     fast = head;
-//   while (fast !== null && fast.next !== null) {
-//     fast = fast.next.next;
-//     slow = slow.next;
-//     if (slow === fast) {
-//       return true; // found the cycle
-//     }
-//   }
-//   return false;
-// }
-// -----
-
```

Comment:
Why was this solution removed?

Reply:
I reversed engineered it in the function